sklearn/plot/_confusion_matrix.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from sklearn.metrics import confusion_matrix
+from sklearn.plot import plot_heatmap
+
+
+def plot_confusion_matrix(y_true, y_pred, classes, sample_weight=None,
+                          normalize=False,
+                          xlabel="Predicted Label", ylabel="True Label",
+                          title='Confusion matrix', cmap=None, vmin=None,
+                          vmax=None, ax=None, fmt="{:.2f}",
+                          xtickrotation=45, norm=None):
+    """Plot the confusion matrix as a heatmap. A confusion matrix is computed
+    using `y_true`, `y_pred` and `sample_weights` arguments. Normalization
+    can be applied by setting `normalize=True`.
+
+    Parameters
+    ----------
+    y_true : array, shape = [n_samples]
+        Ground truth (correct) target values.
+
+    y_pred : array, shape = [n_samples]
+        Estimated targets as returned by a classifier.
+
+    classes : list of strings
```

Comment:
I think this should be optional and by default they should be the union of ``y_true`` and ``y_pred``. I think the ``unique_labels`` function should help.